idd/Energy+.idd.in

```diff
@@ -31865,6 +31865,7 @@ ZoneHVAC:FourPipeFanCoil,
         \key CyclingFan
         \key VariableFanVariableFlow
         \key VariableFanConstantFlow
+        \key MultiStageFan
```

Comment:
@mjwitte Definitions are provided in the documentation. The MultiStageFan cycles between stages or cycles at the lowest fan speed.  The control It is more of capacity staging by varying the fan speed. The CyclingFan is purely cycling at all speed unless the load is higher than the maximum speed.